### PR TITLE
docs lint-3

### PR DESCRIPTION
**Pull request by vinaykulk621**
This PR covers lint for `/geometries`.
This PR is ready to be merged

### CODE REVIEW

**Comment by Mugen87** [comment]
Do you mind rebasing the PR? Certain files like `/extras/core/Curve.html` were already updated by #25814.

**Comment by vinaykulk621** [comment]
i have merged this branch with the dev branch.

**Comment by Mugen87** [comment]
Looking good now!

File: docs/api/en/geometries/BoxGeometry.html [hunk 1]
```diff
@@ -1,74 +1,94 @@
...
-		<h2>Methods</h2>
-		<p>See the base [page:BufferGeometry] class for common methods.</p>
-
-		<h2>Source</h2>
-
-		<p>
-			[link:https://github.com/mrdoob/three.js/blob/master/src/[path].js src/[path].js]
-		</p>
-	</body>
+  <head>
```
Comment (Mugen87) [inline docs/api/en/geometries/BoxGeometry.html:3]: The formatting in this file does not look right. It seems hard tabs have been accidentely replaced with soft tabs.

Comment (vinaykulk621) [reply docs/api/en/geometries/BoxGeometry.html]: Fixed

**Pull request merged**